topics/imaging/tutorials/process-image-bioimageio/tutorial.md

```diff
@@ -0,0 +1,182 @@
+
+Each model in BioImage.IO is tailored for a specific biological task — for example, segmenting nuclei, detecting mitochondria, or identifying neuronal structures — and trained on specific imaging modalities such as electron or fluorescence microscopy {% cite vonChamier2021 GomezdeMariscal2021 %}.
+
+This tutorial will guide you through the process of applying one of these BioImage.IO models to an input image using Galaxy {% cite Afgan2018 %}. You will learn how to upload and configure the model, set the correct input parameters, and interpret the output files.
+
+⚠️ Currently, only the following 11 PyTorch-based BioImage.IO models are compatible with the Galaxy tool *“Process image using a BioImage.IO model with PyTorch”*:
+
+- PlatynereisEMnucleiSegmentationBoundaryModel.pt
+- PlatynereisEMcellsSegmentationBoundaryModel.pt
+- livecellsegmentationboundarymodel.pt
+- HyLFM-Net-stat.pt
+- 3DUNetArabidopsisApicalStemCells.pt
+- CovidIFCellSegmentationBoundaryModel.pt
+- nucleisegmentationboundarymodel.pt
+- HPANucleusSegmentation.pt
+- NeuronSegmentationInEM.pt
+- HPACellSegmentationModel.pt
+- MitochondriaEMSegmentationBoundaryModel.pt
+
+## Available BioImage.IO models in Galaxy
+
+| Model name | Task | Imaging modality | Sample / species | Link |
+|------------|------|------------------|------------------|------|
+| PlatynereisEMnucleiSegmentationBoundaryModel | Nuclei segmentation | Electron microscopy | Platynereis | [View model](https://bioimage.io/#/?id=platynereisemnucleisegmentationboundarymodel) |
+| PlatynereisEMcellsSegmentationBoundaryModel | Cell segmentation | Electron microscopy | Platynereis | [View model](https://bioimage.io/#/?id=platynereisemcellssegmentationboundarymodel) |
+| livecellsegmentationboundarymodel | Live cell segmentation | Phase-contrast Microscopy | Various cell types | [View model](https://bioimage.io/#/?id=livecellsegmentationboundarymodel) |
+| HyLFM-Net-stat | Light field reconstruction | Light field and Fluorescence light microscopy | Zebrafish | [View model](https://bioimage.io/#/?id=hylfm-net-stat) |
+| 3DUNetArabidopsisApicalStemCells | Stem cell segmentation | Confocal / light sheet | Arabidopsis root | [View model](https://bioimage.io/#/?id=3dunetarabidopsisapicalstemcells) |
+| CovidIFCellSegmentationBoundaryModel | Cell segmentation | Fluorescence light microscopy | Infected human cells | [View model](https://bioimage.io/#/?id=covidifcellsegmentationboundarymodel) |
+| nucleisegmentationboundarymodel | Nucleus segmentation | Fluorescence light microscopy| Generic / various | [View model](https://bioimage.io/#/?id=nucleisegmentationboundarymodel) |
```

Comment:
For coherence with the formatting below, maybe:
```suggestion
| NucleiSegmentationBoundaryModel | Nucleus segmentation | Fluorescence light microscopy| Generic / various | [View model](https://bioimage.io/#/?id=nucleisegmentationboundarymodel) |
```
Maybe also put the `🧬` icon as a prefix in front of each model name, so the connection between these and the example below becomes visually immediate.